### PR TITLE
xtest: add case 4010

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Adds test case 4010 which implements a negative test with a too large
key for a secret key object.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

Depends on https://github.com/OP-TEE/optee_os/pull/816

### CODE REVIEW

File: host/xtest/xtest_4000.c [hunk 1]
```diff
@@ -150,6 +151,17 @@ ADBG_CASE_DEFINE(XTEST_TEE_4009, xtest_tee_test_4009,
 		"Description of how to implement ..."
 		 );
 
+ADBG_CASE_DEFINE(XTEST_TEE_4010, xtest_tee_test_4010,
+		/* Title */
+		"Test TEE Internal API create transient object (negative)",
+		/* Short description */
+		"Short description ...",
+		/* Requirement IDs */
+		"TEE-??",
```
Comment (jbech-linaro) [inline host/xtest/xtest_4000.c]: We should do something about this sooner or later, this stuff is just there, but we never use it.

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
Tags applied

**Comment by d3zd3z** [comment]
This shouldn't have been merged until the dependent patch has been merged. I'm seeing that it causes a nice panic down in the TA and ends up bringing down the whole kernel.

**Comment by jbech-linaro** [comment]
@d3zd3z , true, my bad. I thought https://github.com/OP-TEE/optee_os/issues/816 was ready to be merged since I reviewed it the day before and was just waiting for a tags/rebase. We have fixes for this so it will be resolved today.

**Pull request merged**